src/components/Article/Article.jsx

```diff
@@ -1,13 +1,38 @@
-import { Link } from 'react-router-dom'
+import { patchArticleVotesByID } from "../../api";
+import { useState } from 'react';
 
 function Article({article}) {
+
+    const articleCopy = {...article}
+    let [votes, setVotes] = useState(articleCopy.votes)
+    let [err, setErr] = useState(false)
+
+    const handleVote = (event) => {
+        setErr(false)
+        event.preventDefault();
+        let voteType = 0;
+        event.target.name === "positive" ? voteType = 1 : voteType = -1;
+        setVotes(votes += voteType)
+
+        patchArticleVotesByID(article.article_id, voteType)
+        .then((totalVotes) => {
+            setVotes(totalVotes)
```

Comment:
Using the totalvotes that comes from the server is an issue, because if there were multiple users on the site at once who voted at the same time, they'd see the votes jump! For this reason, we're not really interested in what the votes truly are on the server after we initially load the page - we can completely skip this .then

This also makes the count look a bit jumpy when you click the button multiple times, as your local version will go out of sync with the server